java/vector/src/test/java/org/apache/arrow/vector/TestFixedSizeListVector.java

```diff
@@ -265,6 +267,26 @@ public void testUnionFixedSizeListWriter() throws Exception {
     }
   }
 
+  @Test
```

Comment:
it doesn't look like there is a check to ensure writing succeeds?  could you also add one here?

Reply:
sure, fixed.